sql_server/pyodbc/base.py

```diff
@@ -201,7 +201,7 @@ def __init__(self, *args, **kwargs):
                     ops[op] = '%s COLLATE %s' % (sql, collation)
             self.operators.update(ops)
 
-        self.features = DatabaseFeatures(self)
+        self.features = DatabaseFeatures(self, opts.get('supports_nullable_unique_constraints', False))
```

Comment:
i just wanted to overwirte `BaseDatabaseFeatures.supports_nullable_unique_constraint`

how would you think about :
```python
self.features = DatabaseFeatures(self)
self.features.supports_nullable_unique_constraints = opts.get('supports_nullable_unique_constraints', False)
```